src/utils/getUserProfile.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * @flow
+ */
+
+import { get } from 'immutable';
+import { LangUtils } from 'lattice-utils';
+
+import type { UserProfile } from '../types';
+
+const { isNonEmptyString } = LangUtils;
+
+export default function getUserProfile(user :any) :UserProfile {
+
+  const auth0UserProfile = get(user, 'profile', user);
+
```

Comment:
confused about this: the fallback value is the same collection from which we are trying to read?

Reply:
Correct. `user` could be one of multiple objects. If it has `'profile'`, then it's `com.openlattice.organization.OrganizationMember`. If it doesn't, then it's likely going to be `com.openlattice.directory.pojo.Auth0UserBasic`.

Note: this should be moved to `lattice-utils`.